src/calva-fmt/src/config.ts

```diff
@@ -55,10 +72,6 @@ function readConfiguration() {
     }
 }
 
-export function setLspFormatConfig(config: string) {
-    lspFormatConfig = config;
-}
-
 export function getConfig() {
     const config = readConfiguration();
     return config;
```

Comment:
Suggestion: As the `readConfiguration` return a promise, we need to await the config here, right?
```suggestion
export async function getConfig() {
    const config = await readConfiguration();
    return config;
```

Reply:
Yes, maybe that is better than sprinkling all those awaits around like I have done. Thanks!